labcore/analysis/plotting/holo.py

```diff
@@ -466,6 +476,77 @@ def load_data(self) -> DataDict:
         raise NotImplementedError
 
 
+class LoaderNodePath(LoaderNodeBase):
+    """A node that loads data from a specified file location.
+
+    the panel of the node consists of UI options for loading and pre-processing.
+
+    """
+
+    def __init__(self, *args: Any, **kwargs: Any):
+        """Constructor for ``LoaderNodePath``.
+
+        Parameters
+        ----------
+        *args:
+            passed to ``Node``.
+        **kwargs:
+            passed to ``Node``.
+        """
+        super().__init__(*args, **kwargs)
+
+        self.pre_process_opts = RBG(
+            options=[None, "Average"], value="Average", name="Pre-processing"
+        )
+        self.pre_process_dim_input = pn.widgets.TextInput(
+            value="repetition", name="Pre-process dimension"
+        )
```

Comment:
Why are you repeating things from the base class here? Does it have to do with how the items get placed when you render the widgets?

Reply:
You are absolutely right, some of those can be deleted, but some needs to stay for the format of the widgets